measurement/measures/geometry.py

```diff
@@ -103,7 +103,7 @@ def __new__(mcs, name, bases, attrs):
     @staticmethod
     def square(klass):
         for name, unit in klass._units.items():
-            qs_unit = Unit(factor=unit.factor ** 2)
+            qs_unit = Unit(factor=unit.factor**2)
```

Comment:
this is irrelevant to the issue.

Reply:
yes, but CI fails. It's in a separate commit because of that